yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -179,8 +179,8 @@ class MapOperator : public FlowableOperator<U, D, MapOperator<U, D, F>> {
       : Super(std::move(upstream)), function_(std::move(function)) {}
 
   void subscribe(Reference<Subscriber<D>> subscriber) override {
-    Super::upstream_->subscribe(
-        make_ref<Subscription>(this->ref_from_this(this), std::move(subscriber)));
+    Super::upstream_->subscribe(make_ref<Subscription>(
```

Comment:
In some places, we only pass `this->ref_from_this(this)` without calling make_ref<Subscription>. 
i.e.: RequestResponseRequester::subscribe(..) method
Why do we need to call make_ref<Subscription>(.. ?

Reply:
In this instance, `this` refers to a `Flowable`.  We're making a new `Subscription+Subscriber` object here with `make_ref` to pass to the upstream `Flowable` because we're creating a subscription.

`RequestResponderRequester` is different because it's not really a `Flowable`.  It has subscription-level lifetime for an RSocket stream.